modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v2/pages/additional_content.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module PdfConstruction
+    module NoticeOfDisagreement::V2
+      module Pages
+        class AdditionalContent
```

Comment:
I love that. It's just not what we've been doing. In fact I renamed what we had from `additional issues` to `content` because it was more than just additional contestable issues. I'm happy to make the change, but let's do it in another ticket!